### ISSUE TITLE
Cannot install the package for Julia 1.0.1

### ISSUE THREAD

**Issue by kavir1698** [BioJulia/GeneticVariation.jl#22]
Hi,

Based on the README of the package, I expected it to be available on Julia 1.0.1, but it is not in the package manager.
 
## Current Behavior
I get the following error:

```
ERROR: Unsatisfiable requirements detected for package GeneticVariation [9bc6ac9d]:
 GeneticVariation [9bc6ac9d] log:
 ├─possible versions are: [0.1.0, 0.2.0, 0.3.0-0.3.2] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0, 0.2.0, 0.3.0-0.3.2]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```

## Your Environment

- Operating System and version (desktop or mobile): Julia on WSL 17763.55, and Julia 1.0.1 for Windows

  [7e6ae17a] BioSequences v1.1.0
  [a93c6f00] DataFrames v0.14.1
  [e30172f5] Documenter v0.19.6
  [b6b21f68] Ipopt v0.4.2
  [4076af6c] JuMP v0.18.4
  [ee78f7c6] Makie v0.9.0

**Comment by CiaranOMara** [comment]
Should be fixed by #31. Please feel free to reopen.

**Comment by rasmushenningsson** [comment]
There is also [VariantCallFormat.jl](https://github.com/rasmushenningsson/VariantCallFormat.jl), which is based on GeneticVariation.jl.
If you only need the VCF functionality from GeneticVariation.jl, I would recommend using VariantCallFormat.jl instead.
